scapy/layers/can.py

```diff
@@ -0,0 +1,36 @@
+# This file is part of Scapy
+# See http://www.secdev.org/projects/scapy for more informations
+# Copyright (C) Philippe Biondi <phil@secdev.org>
```

Comment:
I think that you should write your name here =)

Reply:
I'm giving the (c) to @phil777 :)

Reply:
Too kind, or too lazy ? =)